### PR TITLE
add check that fps is at most 200

### LINKED ISSUE #663: FPS of video in output.media_writer can exceed the maximum supported FPS by MPEG 4
**Description**
In [this line](https://github.com/aimakerspace/PeekingDuck/blob/main/peekingduck/pipeline/nodes/output/media_writer.py#L103)  of code, if `saved_video_fps` is above 65535, `cv2.VideoWriter` throws the following error:
`timebase 1/90000 not supported by MPEG 4 standard, the maximum admitted value for the timebase denominator is 65535`

### PR DESCRIPTION
**Pull request by JamesSng**
Fixes #663

**Change Made**
Add check that FPS is at most 200 so that invalid FPS values will not get passed in

### CODE REVIEW

**Comment by JamesSng** [comment]
Instead of setting the FPS to be the minimum of the FPS returned by OpenCV at 200, I use ```self.saved_video_fps``` if FPS is more than 200 instead. The first method would cause the output video to be sped up by a lot.

File: peekingduck/pipeline/nodes/input/visual.py [hunk 1]
```diff
@@ -242,7 +242,9 @@ def _get_next_frame(self) -> Dict[str, Any]:
             "img": None,
             "filename": self._file_name if self._file_name else self.filename,
             "pipeline_end": True,
-            "saved_video_fps": self._fps if self._fps > 0 else self.saved_video_fps,
+            "saved_video_fps": self._fps
+            if (self._fps > 0 and self._fps <= 200)
```
Comment (ongtw) [inline peekingduck/pipeline/nodes/input/visual.py:246]: Can recode `(self._fps > 0 and self._fps <= 200)` in the pythonic way `(0 < self._fps <= 200)`.

**Review by ongtw — COMMENTED** [review]
Suggested one change.

File: peekingduck/pipeline/nodes/input/visual.py (hunk 1 above)
Comment (JamesSng) [reply peekingduck/pipeline/nodes/input/visual.py]: fixed

**Pull request merged**